assets/css/header-navbar.css

```diff
@@ -0,0 +1,153 @@
+body{
+    position: relative;
+}
+main{
+    background-color: #f9f7ff;
+}
+header{
+    height:96.5vh;
+    border-radius: 38px;
+    width:90%;
+    margin:0 auto;
+    box-sizing: border-box;
+    padding-top: 1rem !important;
```

Comment:
if page has many headers and u use important it is a bad for ur friend in team so using BEM to give it classes